lib/src/utils.dart

```diff
@@ -43,3 +43,18 @@ Patch patchAfter(AstNode node, String text) {
 void warn(String message, FileSpan context) {
   print(context.message("WARNING - $message"));
 }
+
+class MigrationException {
+  final String message;
+  final FileSpan context;
```

Comment:
In most places, we just call the span associated with a class `span`.

Also, document this.